packages/sit-onyx/src/composables/useCustomValidity.ts

```diff
@@ -224,3 +225,16 @@ export const useCustomValidity = (options: UseCustomValidityOptions) => {
     errorMessages,
   };
 };
+
+const formatMinMax = (
+  type: UseCustomValidityOptions["props"]["type"],
+  value?: DateValue,
+): string | undefined => {
+  if (!type || !["date", "datetime-local"].includes(type)) return value?.toString();
+
+  const date = value != undefined ? new Date(value) : undefined;
+  if (!isValidDate(date)) return value?.toString();
+
+  if (type === "date") return date.toLocaleDateString();
+  else if (type === "datetime-local") return date.toLocaleString();
```

Comment:
Are we able to pass the local to `toLocaleDateString` and `toLocaleString`?

Reply:
Done